public/js/dom.js

```diff
@@ -0,0 +1,44 @@
+// const url = `https://api.themoviedb.org/3/search/movie?api_key=${config.apiKey }&language=en-US&query=`;
+function selecElement(id) {
+	return document.querySelector(id);
+}
+function addEventListenerForSelector(selectorElement, event, callback){
+	return document.querySelector(selectorElement).addEventListener(event,callback);
+}
+function createElement(e){
+	return document.createElement(e);
+}
+addEventListenerForSelector('#submit-btn', 'submit', (event) => {
+	event.preventDefault();
+	if (selecElement('#searchInput').value == ""){
```

Comment:
I prefer to save the element.value in const after that compare the constant with empty string
and use triple equals instead of double equals in comparing  
like this: 
```
const query = selectElement('#searchInput').value.trim();
if ( query  ===  "" )
  alert(' Enter a movie name');
```